simapp/sim_bench_test.go

```diff
@@ -2,57 +2,46 @@ package simapp
 
 import (
 	"fmt"
-	"io/ioutil"
 	"os"
 	"testing"
 
+	"github.com/cosmos/cosmos-sdk/x/simulation"
 	abci "github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/libs/log"
-	dbm "github.com/tendermint/tm-db"
-
-	"github.com/cosmos/cosmos-sdk/simapp/helpers"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/simulation"
 )
 
 // Profile with:
 // /usr/local/go/bin/go test -benchmem -run=^$ github.com/cosmos/cosmos-sdk/simapp -bench ^BenchmarkFullAppSimulation$ -Commit=true -cpuprofile cpu.out
 func BenchmarkFullAppSimulation(b *testing.B) {
-	logger := log.NewNopLogger()
-	config := NewConfigFromFlags()
-	config.ChainID = helpers.SimAppChainID
+	config, db, dir, _, _, err := SetupSimulation("goleveldb-app-sim", "Simulation")
+	if err != nil {
+		fmt.Println(err, "simulation setup failed")
+		b.Fail()
```

Comment:
Use `FailF` instead of printing before hand.

Reply:
I think you meant `Fatalf` and `Fatal`. I updated the tests